package.json

```diff
@@ -74,6 +74,7 @@
     "@types/string-template": "^1.0.2",
     "@types/unzipper": "^0.10.3",
     "babel-loader": "^8.1.0",
+    "babel-plugin-inline-import-data-uri": "^1.0.1",
```

Comment:
what is inline-import-data-uri for?

Reply:
oh does this make it so that our usage of like import image from 'image.png' works from everywhere without special loaders?

Reply:
> oh does this make it so that our usage of like import image from 'image.png' works from everywhere without special loaders?

Basically, yes, it makes images into inline data-uris so the consuming code doesn't have to worry about find/copying the images to somewhere the component can find them.